### PR TITLE
fixes #420 smooth animation

### LINKED ISSUE #420: Smooth Transition of Instruction section
**Current State of the Meshery.io site**
Instructions Section is neither opening smoothly nor closing. 

**Desired State of the Meshery.io site**

Smooth opening/closing of Instructions section in [meshery.io](https://meshery.io/)

https://user-images.githubusercontent.com/44670961/113330128-08ce7100-933c-11eb-9a0e-241b02572d81.mp4

---
**Contributor Resources**
- See the [Contributing to Meshery.io Website](https://github.com/layer5io/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md.

### PR DESCRIPTION
**Pull request by rijusougata13**
**Description**

This PR fixes #420 

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by rijusougata13** [comment]
@leecalcote  PTAL

**Comment by Nikhil-Ladha** [comment]
Hi, @rijusougata13 please follow the steps [here](https://github.com/layer5io/meshery.io/pull/425/checks?check_run_id=2313947000) to pass the DCO check.

Also, there is no closing animation but I am not sure If we need one here?
/cc @leecalcote

**Comment by rijusougata13** [comment]
its maybe because when we are closing we are just simply removing showDiv class. So there is no closing transition . But  you wil notice if you switch from different icon there will be a smooth animation

**Comment by rijusougata13** [comment]
DCO check passed @Nikhil-Ladha

**Comment by Nikhil-Ladha** [comment]
> its maybe because when we are closing we are just simply removing showDiv class. So there is no closing transition . 
Closing animation needs to be added separately if needed.

> But you wil notice if you switch from different icon there will be a smooth animation

That's because the other dropdown is getting the opening animation :)

**Comment by rijusougata13** [comment]
@Nikhil-Ladha  added closing animation. Take a look if its ok or not.

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha added closing animation. Please
> Take a look if its ok or not.

Hmm... I didn't ask for the closing animation because it looked fine to me in the previous implementation itself. It was rather a question to Lee and Chinmay for the same 😅

**Comment by Nikhil-Ladha** [comment]
My opinion is we don't need a closing animation.

**Comment by chinmaym07** [comment]
I think previous implementation is better than we have it right now .

**Comment by rijusougata13** [comment]
yeah i implemented for just backup reverting it to prev state

**Review by Nikhil-Ladha — APPROVED** [review]

File: _sass/layout.scss [hunk 1]
```diff
@@ -256,11 +256,21 @@ td {
 	transition: cubic-bezier(0, 1, 0, 1);
 }
 
+
 .hideDiv {
-	display: none;
+	display:hidden;
```
Comment (Nikhil-Ladha) [inline _sass/layout.scss:261]: `hidden` is not a valid value for `display`. Please send out a PR to update this.

**Pull request merged**